apps/examples/wifi_manager_sample/wm_test_usage.h

```diff
@@ -75,6 +75,19 @@
 	"	 wm_test power [mode]\n"				\
 	"	 mode examples : on, off\n\n"
 
+#define WT_INTEROP_USAGE						\
```

Comment:
how about to add file path parameter? I think  it's more useful.

Reply:
add the file path parameter in wm_test interop [file path]